extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/counters/CounterSnapshotsTree.java

```diff
@@ -108,4 +108,14 @@ private void putAll(final Map<Integer, Map<Integer, CounterSnapshots>> otherMap)
       }
     }
   }
+
+  @Override
+  public String toString()
```

Comment:
We don't need this now right?

Reply:
It's needed for the new test added in the PR: `testMSQSelectRunningQueryWithDetail` as `assertSqlStatementResult()` method has the following assertion logic for  counters:
```java
if (actual.getCounters() == null || expected.getCounters() == null) {
       Assert.assertEquals(expected.getCounters(), actual.getCounters());
     } else {
       Assert.assertEquals(expected.getCounters().toString(), actual.getCounters().toString());
     }
```